src/output/textwrap/mod.rs

```diff
@@ -0,0 +1,113 @@
+//! Fork of `textwrap` crate
+//!
+//! Benefits of forking:
+//! - Pull in only what we need rather than relying on the compiler to remove what we don't need
```

Comment:
Are you not already using the optional Cargo features in Textwrap? As per [the documentation](https://docs.rs/textwrap/latest/textwrap/#cargo-features), every Textwrap dependency is optional.

Reply:
I'm not referring to textwrap's optional depednencies but the more general behavior which has more code to it.